backend/prisma/schema.prisma

```diff
@@ -18,3 +31,44 @@ model User {
   failedLogins Int     @default(0)
   verified     Boolean @default(false)
 }
+
+model Category {
+  id        Int        @id @default(autoincrement())
+  name      String     @unique
+  questions Question[]
+}
+
+model Topic {
+  id        Int        @id @default(autoincrement())
+  name      String     @unique
+  questions Question[]
+}
+
+model SolutionTemplate {
+  id         Int      @id @default(autoincrement())
+  language   Language
+  template   String
+  question   Question @relation(fields: [questionId], references: [id])
+  questionId Int
+}
+
+model TestCases {
+  id         Int      @id @default(autoincrement())
+  inputs     String[]
```

Comment:
Any thoughts on whether to store the input as a String or String[]? The sampleTestCase provided in the json is in String format so if we want to transform it to String[], we will need to clean up the json further by splitting along "\n"

Reply:
I think it's best to store it as a `String[]` since we will have to normalize the data some point down the line and its best to do it before the database.